project-metrics/metrics_service/pruning.py

```diff
@@ -0,0 +1,14 @@
+"""Module to provide pruning for metric data tables."""
+
+import logging
+
+import db_engine
+import models
+
+
+def prune() -> None:
```

Comment:
I wouldn't merge this just yet, but keep it around. Once we have some sense of how fast the data grow, we might decide to keep it around a bit longer (so we can, say, display historic data)
Since there's no personal data involved, there's no rush to delete the data

Reply:
Pulling the time logic out of this PR; will roll it into the upcoming PR adding time filtering to metrics. Will close this PR afterwards, not to hard to rewrite `models.Build.last_90_days(db_engine.get_session(), negate=True).delete()`